frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```
@@ -433,9 +434,12 @@ action.GitMachete.OpenMacheteTabAction.notification.message.no-git=Ensure that G
 
 action.GitMachete.CheckoutNextAction.text=Git Machete: Checkout Next Branch
 action.GitMachete.CheckoutNextAction.description=Checkout next branch (''{0}'')
+action.GitMachete.CheckoutNextAction.undefined.branch-name=No branches after (''{0}'') in the machete layout
```

We call it 
```suggestion
action.GitMachete.CheckoutNextAction.undefined.branch-name=No branches after (''{0}'') in the branch layout
```
`machete layout` is clear as well, but let's be consistent